setup.cfg

```
@@ -1,7 +1,7 @@
 [metadata]
 name = eddymotion
 author = Derek Pisner
-author_email = dpisner@utexas.edu
+author_email = dpysalexander@gmail.com
```

Like the other, please pull out on the other PR. I think we should also follow the general nipreps guidelines and write the nipreps account.

```suggestion
author_email = nipreps@gmail.com
```

---

Ditto, see https://github.com/nipreps/eddymotion/pull/119